dev_suites/demo_ig_stu1/groups/demo_group.rb

```diff
@@ -11,7 +11,7 @@ class DemoGroup < Inferno::TestGroup
     # while also allowing type hints at a higher level.
 
     input :url, title: 'URL', description: 'Insert url of FHIR server'
-    input :patient_id, title: 'Patient ID'
+    input :patient_id, title: 'Patient ID', default: 'Default Patient'
```

Comment:
It would be really helpful for development if you set these defaults to our reference serve url and patient id.